common/bin-common/src/build_information/mod.rs

```diff
@@ -156,6 +163,15 @@ pub struct BinaryBuildInformationOwned {
     /// Provides the cargo debug mode that was used for the build.
     // NOTE: keep the old name cargo_profile instead of cargo_debug for backwards compatibility
     pub cargo_profile: String,
+
+    // VERGEN_CARGO_TARGET_TRIPLE
+    /// Provides the cargo target triple that was used for the build.
+    #[serde(default = "unknown")]
+    pub cargo_triple: String,
```

Comment:
So, this thing here will impact backwards compatibility

Reply:
I think it should be fine. since you've added the `default`, old code should still be deserialisable